### ISSUE TITLE
FR: Change Sourcegraph code view away from tables to support screen readers

### ISSUE THREAD

**Issue by francisschmaltz** [sourcegraph/sourcegraph#974]
To support screen readers, we should have an alternative way to display code on Sourcegraph other than a table view. Table views do not work well with screen readers and we should come up with a better solution. 

As we currently use table views for all code layouts, this might be a huge undertaking and we should discuss how/when to move away from tables, or if there is an alternative we can use to solve the problems with screen readers.
